orchestration/src/main/java/com/sap/ai/sdk/orchestration/AzureContentFilter.java

```diff
@@ -0,0 +1,49 @@
+package com.sap.ai.sdk.orchestration;
+
+import static com.sap.ai.sdk.orchestration.client.model.AzureThreshold.fromValue;
+
+import com.sap.ai.sdk.orchestration.client.model.AzureContentSafety;
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+import lombok.experimental.Accessors;
+import lombok.val;
+
+@Data
+@Accessors(fluent = true)
+public final class AzureContentFilter implements ContentFilter {
+  @Nullable private Sensitivity hate;
+  @Nullable private Sensitivity selfHarm;
+  @Nullable private Sensitivity sexual;
+  @Nullable private Sensitivity violence;
+
+  @RequiredArgsConstructor
+  public enum Sensitivity {
+    HIGH(0),
```

Comment:
Aligned w/ Orchestration team, to be aligned w/ JS and Launchpad

Reply:
(Major)

Do you have a reference/link where this assignment 0=high is explained?
I would've assumed these are thresholds with terms high/low easily confusable. The last thing we want is adding confusion with those terms.

Reply:
The terminology isn't consistent yet, which is why this is so confusing, which is why this enum exists. Currently, the [orchestration docs](https://help.sap.com/docs/sap-ai-core/sap-ai-core-service-guide/orchestration-workflow) still use largely numbers, AI Launchpad and the Python SDK use the terms low / medium / high, but with the inverse semantic of high being "high threshold" i.e. allow everything.

Aligning on a consistent terminology is out of scope for this PR, will be done separately. So for now maybe consider the current state a placeholder for whatever will be aligned. Or maybe we should completely remove it from this PR and switch back to plain numbers, until alignment is done. No strong opinion.